sdk/src/lib.rs

```diff
@@ -72,54 +61,16 @@ pub use solana_sdk_macro::pubkeys;
 #[rustversion::since(1.46.0)]
 pub use solana_sdk_macro::respan;
 
-// On-chain program specific modules
-pub mod account_info;
-pub mod entrypoint;
-pub mod entrypoint_deprecated;
-pub mod log;
-pub mod program;
-pub mod program_error;
-
-#[cfg(all(feature = "program", not(target_arch = "bpf")))]
-extern crate lazy_static;
-
-#[cfg(all(feature = "program", not(target_arch = "bpf")))]
-pub mod program_stubs;
-
-// Unused `solana_sdk::program_stubs!()` macro retained for source backwards compatibility with v1.3.x programs
+// Unused `solana_sdk::program_stubs!()` macro retained for source backwards compatibility with older programs
 #[macro_export]
 #[deprecated(
-    since = "1.4.2",
+    since = "1.4.3",
     note = "program_stubs macro is obsolete and can be safely removed"
 )]
 macro_rules! program_stubs {
     () => {};
 }
 
-pub mod serialize_utils;
-
-// Modules not usable by on-chain programs
-#[cfg(feature = "everything")]
```

Comment:
I moved the `#[cfg(feature = "everything")]`s into inner cfgs in each file to clean up lib.rs

Reply:
Though does require searching through all the files to figure out if everything.